app/models/staff_response.rb

```diff
@@ -5,6 +5,8 @@ class StaffResponse
   attr_accessor :visit, :user
 
   before_validation :check_slot_available
+  before_validation :check_lead_visitor_not_banned
+  before_validation :check_lead_visitor_on_list
 
   validate :validate_visit_is_processable
   validate :visit_or_rejection_validity
```

Comment:
@StupidCodeFactory I just came across this method for my work and thought it might be related to this PR since it checks that there is at least one adult. Is it still relevant or do we need to check for valid lead visitor instead?

I'm not 100% sure right now, but I think it might need changing since at least one valid visitor is the old logic